.github/workflows/check-metabolictasks.yml

```
@@ -0,0 +1,19 @@
+name: Test metabolic tasks
+
+on: [push, pull_request]
+
+jobs:
+  check-metabolictasks:
+    runs-on: self-hosted
+    timeout-minutes: 60
+    strategy:
+      matrix:
+        task-type: [essential, verification]
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Check metabolic tasks
```

The task type can also be used in the step name:
```suggestion
    - name: Check ${{ matrix.task-type }} metabolic tasks
```